src/main/java/seedu/address/model/Model.java

```diff
@@ -59,31 +61,64 @@ public interface Model {
      */
     boolean hasPerson(Person person);
 
+    /**
+     * Returns true if the same company is in the address book.
+     *
+     * @param company Company to be checked.
+     * @return true if in address book.
+     */
+    boolean hasCompany(Company company);
+
```

Comment:
For the JavaDocs in general, are we sticking to the existing code base style, or using this style?

Reply:
I think they are the same, since according to the coding standard provided, both `@param` and `@return` can be omitted they are made obvious or already explain in the rest of the comment.

Reply:
I think they are the same, since according to the [coding standard](https://se-education.org/guides/conventions/java/intermediate.html#comments) provided, `@param` and `@return` can be omitted if they are made obvious or explained in the rest of the comments.